### PR TITLE
owner: check the version of the ticdc cluster

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
check the version of the ticdc cluster, make sure all the version of ticdc nodes is consistent

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note

- No release note

### CODE REVIEW

File: cdc/owner/owner.go [hunk 1]
```diff
@@ -106,6 +107,9 @@ func (o *Owner) Tick(stdCtx context.Context, rawState orchestrator.ReactorState)
 	ctx := stdCtx.(cdcContext.Context)
 	state := rawState.(*model.GlobalReactorState)
 	o.updateMetrics(state)
+	if !o.clusterVersionConsistent(state.Captures) {
+		return state, nil
```
Comment (asddongmen) [inline cdc/owner/owner.go:113]: may be can sleep 1s here to avoid excessive output of log

File: cdc/owner/owner.go [hunk 2]
```diff
@@ -106,6 +107,11 @@ func (o *Owner) Tick(stdCtx context.Context, rawState orchestrator.ReactorState)
 	ctx := stdCtx.(cdcContext.Context)
 	state := rawState.(*model.GlobalReactorState)
 	o.updateMetrics(state)
+	if !o.clusterVersionConsistent(state.Captures) {
+		// sleep one second to avoid to print too much log
```
Comment (liuzix) [inline cdc/owner/owner.go:111]:
```suggestion
		// sleep one second to avoid printing too much log
```

**Comment by liuzix** [comment]
rest /lgtm

**Review by liuzix — APPROVED** [review]

**Review by ben1009 — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Pull request merged**